client/components/screen-options-tab/index.js

```diff
@@ -60,8 +68,19 @@ const ScreenOptionsTab = () => {
 		return null;
 	}
 
+	// Only visible on single-site screens of WordPress.com Simple and Atomic sites.
+	if ( ! wpAdminPath || ! siteId || ( isJetpack && ! isAtomic ) ) {
+		return null;
+	}
+
+	const onSwitchView = ( view ) => {
+		if ( view === 'default' ) {
```

Comment:
Seeing as this string value isn't typed due to us not having TS here, and is used elsewhere along with `classic` do you think it's worth creating some more descriptive exported `const` values for them?

```js
export const CALYPSO_SCREEN_VIEW = 'default'
export const WPADMIN_SCREEN_VIEW = 'classic'
```

Reply:
It will likely give these values more context, as I had to find its usage to be sure.

Reply:
https://github.com/Automattic/wp-calypso/pull/53866/commits/fe8200786259ae853c031375b29054aa5aa9346d